frontend/src/metabase/query_builder/components/view/sidebars/QuestionInfoSidebar/tests/common.unit.spec.tsx

```diff
@@ -0,0 +1,83 @@
+import userEvent from "@testing-library/user-event";
+import {
+  createMockCard,
+  createMockModerationReview,
+} from "metabase-types/api/mocks";
+import { screen } from "__support__/ui";
+import { setup } from "./setup";
+
+describe("QuestionInfoSidebar", () => {
+  describe("description", () => {
+    it.each([
+      createMockCard({ name: "Question", description: "abc", dataset: false }),
+      createMockCard({ name: "Model", description: "abc", dataset: true }),
+    ])("should display description of a $card.name", async card => {
+      await setup({ card });
+      expect(screen.getByText(card.description ?? "")).toBeInTheDocument();
```

Comment:
I understand `?? ""` is there to satisfy TypeScript, but do you know if it's possible to TypeScript to know `description` in this case, won't be `null` since we supply both card a `description`?

Reply:
> I understand `?? ""` is there to satisfy TypeScript, 

If this is true, then `!` would have been more appropriate as it communicates the intention better:

```suggestion
      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
      expect(screen.getByText(card.description!)).toBeInTheDocument();
```

Reply:
Moved description to a constant